escrow/tests/cancel.rs

```diff
@@ -0,0 +1,73 @@
+mod utils;
```

Comment:
I'd import `utils` as public to prevent `function is never used` warnings.
```suggestion
pub mod utils;
```